blocks/components/editable/dom-to-react.js

```diff
@@ -0,0 +1,29 @@
+import { createElement } from 'react';
```

Comment:
Since we'll need this in [`blocks/api/query.js`](https://github.com/WordPress/gutenberg/blob/master/blocks/api/query.js) as well, we'll probably want to place the file somewhere more generic. Kinda ties to #408, as it seems in addition to component components, we'll need common utilities as well.

That, or we could just commit to individually publishing npm modules for the utilities we create.